app/views/admin/infobase/_info.html.haml

```diff
@@ -0,0 +1,6 @@
+.ui.tab.segment{"data-tab": "users", id: 'users-tab-segment'}
+  = render 'registered_users', infobase: infobase
+.ui.active.tab.segment{"data-tab": "menu", id: 'menu-tab-segment'}
```

Comment:
`.ui.active.tab.segment#menu-tab-segment{ data: { tab: 'menu'  } }`
1. forget about `""`
2. if you HAVE to use string key `'data-tab': 'menu'`
3. you better think twice if you "have" to use string key